### PR TITLE
Add Standard Fonts Support

### PR DESCRIPTION
**Pull request by jrbostic**
These were the monospaced fonts I included:
https://fontlibrary.org/en/font/courier-code
https://fontlibrary.org/en/font/courier-prime
https://fontlibrary.org/en/font/anonymous-pro
https://fontlibrary.org/en/font/gnutypewriter

Changes are pretty minor.  

I formatted some of the config strings because mixed `"` and `' `were bothering me.

For testing, I may only be able to cover the new config param... will follow up with that soon.

No custom user-defined fonts at this point.

### CODE REVIEW

File: lib/fountain.coffee [hunk 1]
```diff
@@ -18,122 +18,128 @@ module.exports = Fountain =
...
+          type: 'boolean'
+          default: true
+          title: 'Double space between scenes'
+          description: 'Whether a double space should be inserted between scenes in pdf'
     settings5:
-      title: "Custom"
+      title: 'Custom'
       type: 'object'
       properties:
+        font_family:
```
Comment (jrbostic) [inline lib/fountain.coffee]: This is the only new config.

File: package.json [hunk 2]
```diff
@@ -19,7 +19,7 @@
     "atom-space-pen-views": "^2.0.0",
     "underscore-plus": "^1.0.0",
     "xregexp": "^3.1.1",
-    "afterwriting": "^1.7.0",
+    "afterwriting": "^1.10.0",
```
Comment (jrbostic) [inline package.json]: Apparently the `^` actually upgrades the minor version on its own when running `npm i`, but it never hurts to be clear.

**Comment by superlou** [comment]
Do you want to add font-size configuration in this pull request?

I'm fine with standardizing the single/double quotes on the configuration items. I think I'd like to extract them to a separate file anyway in a later commit.

With respect to licensing, my understanding is that 1 font uses the Bitstream Vera license which allows embedding with no mention of attribution. The remaining 3 use the SIL Open Font License which also [does not require attribution](http://scripts.sil.org/cms/scripts/page.php?item_id=OFL-FAQ_web#e71fabc0).

**Comment by jrbostic** [comment]
I don't think font size is going to be something that we can manipulate.  First reason is because there's no exposure of that setting in the afterwriting configuration. The other is based on the discussion I had with afterwriting author; there are some hardcoded heights and widths in the PDF generation code in afterwriting (I think he said it was related to title page layout).  So, changes would need to start in the afterwriting project.

Are you concerned about the bitstream Vera license?  Like I said, I'm open to excluding or including any suggested fonts.  It would also be nice if we could find some way to attribute just for the sake of posterity and whatnot, but I haven't thought about it much.

**Comment by superlou** [comment]
Acknowledge on the font-size issue.

IANAL, but not worried about the Vera license. That one is a lot simpler and also allows sharing without attribution. I'm not against crediting the authors, and agree that would be good to do, just trying to avoid accidentally breaking any licensing rules. For all 4 of those, I believe the limitations relevant to us are that we can't strip the license embedded in the font.

**Comment by jrbostic** [comment]
Okay, maybe we could just list supported fonts in the docs, with links to their pages on that open font website?

**Comment by jrbostic** [comment]
@superlou this is finished from my point of view.  Any additional concerns with it?

**Comment by superlou** [comment]
Functionality-wise, it seems good. I'm getting 3 failures in the test suit, but I also get those on the current master. Merging it in.

I was hoping to release this in the next minor version along with the switch to Docks. Unfortunately, I haven't had time to really work on it, and while it basically works, there are still serialization edge cases. If this PR resolves the core Font functionality you wanted to add, let me know and I'll do a minor release ASAP.

**Comment by jrbostic** [comment]
I'm fine with waiting until you're ready to release the dock stuff.  I'll take a look at the failing tests on master as well.  We should at least resolve those before a release... for good form.

**Comment by jrbostic** [comment]
@superlou I'm not seeing any failing tests on current master.  Is it possibly a couple flaky tests or do you have any untracked files that might be causing it?

**Comment by jrbostic** [comment]
That test doesn't use the default config, it sets values and retrieves them.  The one thing about this test is it's using asynchronous Jasmine waiting structures, so it could get thrown off  if it's not properly resolving in time.  It's possible that the wait duration is too short in certain contexts... For some reason.  Like, if a computer was overtaxed for some reason, it may fail like this.  You could try increasing the wait duration.

**Comment by superlou** [comment]
I tried increasing the wait duration to 10 seconds without any change. I'll have to dig into this more.

**Pull request merged**